Userland/Libraries/LibWasm/Parser/Parser.cpp

```diff
@@ -254,10 +254,10 @@ ParseResult<BlockType> BlockType::parse(Stream& stream)
     ReconsumableStream new_stream { stream };
     new_stream.unread({ &kind, 1 });
 
-    auto index_value_or_error = stream.read_value<LEB128<ssize_t>>();
+    auto index_value_or_error = new_stream.read_value<LEB128<i32>>();
```

Comment:
🤔 what about the last bit in the i33?

Reply:
Hmm wait... I'm not really sure... I think it would get ignored. Looking at [WABT](https://github.com/WebAssembly/wabt/blob/2c50f59f518968643a36e800c0268bcfabc485ac/src/binary-reader.cc#L350), they just parse i32's. I don't think that's spec compliant, though. I could either try to fix it, or we could just ignore it and put a FIXME. What do you recommend?

Reply:
Fixme sounds good.